src/components/reference-book/page.cjsx

```diff
@@ -13,7 +13,7 @@ GetPositionMixin = require '../get-position-mixin'
 {ReferenceBookStore} = require '../../flux/reference-book'
 {ReferenceBookExerciseStore} = require '../../flux/reference-book-exercise'
 
-EXERCISE_MATCHER = new RegExp('#ost\/api\/ex\/(.*)')
+EXERCISE_MATCHER = new RegExp('\/exercises\/(.*)')
```

Comment:
Not very useful for you I guess... The urls now look like `https://exercises-dev.openstax.org/api/exercises?q=tag%3Aapbio-ch02-ex026`.